pkg/cmd/pulumi/policy_new.go

```diff
@@ -189,9 +190,16 @@ func runNewPolicyPack(args newPolicyArgs) error {
 func installPolicyPackDependencies(proj *workspace.PolicyPackProject, projPath, root string) error {
 	// TODO[pulumi/pulumi#1334]: move to the language plugins so we don't have to hard code here.
 	if strings.EqualFold(proj.Runtime.Name(), "nodejs") {
-		if bin, err := nodeInstallDependencies(); err != nil {
+		fmt.Println("Installing dependencies...")
```

Comment:
So we need to use logging instead of fmt.Println?

Reply:
So this is copied from nodeInstallDependencies which already used fmt.Println

Reply:
It's moving processes though right. The plugin executable runs in a process where stdout is used to communicate the port on which the engine is running. Printing after that might work. I'm just careful. Hopefully the tests cover us here.

Reply:
Nope this hasn't moved process

Reply:
Ah never mind. Perfect ty. pkg/cmd you're right.